src/test/resources/features/ceva.feature

```diff
@@ -0,0 +1,75 @@
+Feature:
+
+@Data
+  Scenario:
+    Given I go to "http://localhost:3000/"
+    When I click on Login
+    Then I am on "http://localhost:3000/dashboard"
+    And I check for search
+
+
+ @MRA-201a, @MRA-201
```

Comment:
tagurile astea nu sunt mai usor sa fie depistate daca stiu ce este scenariul respectiv ? 
gen "login" in loc de MRA?     MRA'urile nu o sa le stim pe derost